From: "Neal H. Walfield" <neal@g10code.com>
Date: Fri, 6 Jan 2017 11:51:08 +0100
Subject: gpg: Remove period at end of warning.

* g10/tofu.c (tofu_register_encryption): Remove period at end of
warning.

Signed-off-by: Neal H. Walfield <neal@g10code.com>
(cherry picked from commit 6f9d8a956b2ca0f5a0eb7acc656fc17af2f2de47)
---
 g10/tofu.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/g10/tofu.c b/g10/tofu.c
index 8d535fa6c..149a18545 100644
--- a/g10/tofu.c
+++ b/g10/tofu.c
@@ -3480,7 +3480,7 @@ tofu_register_encryption (ctrl_t ctrl,
 
       if (! user_id_list)
         log_info (_("WARNING: Encrypting to %s, which has no "
-                    "non-revoked user ids.\n"),
+                    "non-revoked user ids\n"),
                   keystr (pk->keyid));
     }
 
